website/versioned_docs/version-17.3.0/ophash_before_injecting.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Getting the Operation Hash without Injecting
+id: ophash_before_injecting
+author: Davis Sawali
+---
+
+This section serves as a guide to get the operation hash without injecting it into the blockchain.
+
+## Getting the Operation Hash
+
+There might come a time when you, the developer, would like to grab the operation hash without injecting it into the blockchain.
+
+To accomplish that, we have a utility function called `encodeOpHash()` in the `@taquito/utils` package. This function takes in the signed operation bytes in the form of a `string`, and outputs the operation hash of said transaction without injecting it.
+
+### Usage example
+
+Here is a simple example on getting the operation hash of a transaction (transfer) operation:
+
+```js
+
+// import { TezosToolkit } from '@taquito/taquito';
+// import { encodeOpHash } from '@taquito/utils';
+// import { forger } from '@taquito/local-forging';
+
+// const Tezos = new TezosToolkit('RPC_ENDPOINT');
+
+const preparedTransfer = await Tezos.prepare.transaction({
+  amount: 1,
+  to: 'tz1ZfrERcALBwmAqwonRXYVQBDT9BjNjBHJu'
+});
+
+
+// forge the transaction operation
+const forgeParams = await Tezos.prepare.transaction(prepared);
```

Comment:
think we need to choose either line 27 to be `prepared` or here to be `preparedTransfer`
For our lazy user if they copy paste and will just work